docs/classes/ERC725.md

```diff
@@ -754,7 +754,8 @@ ERC725.encodePermissions({
   ENCRYPT: false,
   DECRYPT: false,
   SIGN: false,
-  EXECUTE_RELAY_CALL: false
+  EXECUTE_RELAY_CALL: false,
+  "4337_PERMISSION": false
```

Comment:
It's not good practice to have property names start with numeric. ERC4337_PERMISSION would be a lot better to deal with in javascript because it's a standard identifier.